test/unit/lib.js

```diff
@@ -403,3 +403,22 @@ test('should loop through each element of an array', function() {
     }
   });
 });
+
+//getFileExtension tests
+test('should get the file extension of the passed path', function() {
```

Comment:
Can we add a test for a full URL? e.g. `http://...` I can see that it supports it but it'd be good to have a test for it.